### DISCUSSION TITLE
Alpha 16

### DISCUSSION THREAD

**Discussion by cameronwhite** [powertab/powertabeditor#347] [General]
### Installing

- For Windows and Mac users, download the installer from the `Assets` section at the bottom of the release notes.
- For Linux users, a [Snap package](https://snapcraft.io/powertabeditor) is now available. A Flatpak package is in progress and will be available in the near future.
 
### Added
- Added support for tremolo bars (#8).
- .pt2 files are now 3-4x smaller in file size.
- The Windows installer has been revamped. It is recommended to first uninstall the existing version before upgrading.
- For Linux users, the application can now be easily installed as a Snap package (https://snapcraft.io/powertabeditor).
- The macOS installers are now signed and notarized. This resolves the "developer cannot be verified" warnings when running for the first time.
- The application can now be [translated into other languages](https://hosted.weblate.org/engage/powertabeditor/).

### Changed
- Removed dependency on boost::filesystem. Instead, std::filesystem (C++17) is now used. See the README for updated build instructions.
- Removed dependency on RapidJSON with nlohmann-json. See the README for updated build instructions.

### Fixed
- Fixed an issue where stopping MIDI playback while a "let ring" was active could incorrectly keep the "let ring" active when restarting playback from the beginning (#337).
- Fixed a bug where the mixer's volume settings could be ignored when starting MIDI playback partway through the score (#335).
- Fixed an issue where the score's title could be scaled incorrectly when printed (#338).
- Fixed potential crashes on exit when the tuning dictionary failed to load (#342).
- Fixed a bug that could cause player changes to be incorrect created from .ptb files. This could lead to later issues with MIDI playback.
- Fixed issues where .ptb files containing multiple text items at the same position did not import correctly.
- Fixed a bug in the score polisher that could multiple notes to end up at the same position. This could lead to later issues with editing and MIDI playback.
- Improvements for how the first barline in a system is rendered (#2).

### Credits
Thanks to the following contributors who worked on this release:
- @cameronwhite

<hr /><em>This discussion was created from the release <a href='https://github.com/powertab/powertabeditor/releases/tag/2.0.0-alpha16'>Alpha 16</a>.</em>

**Comment by Ebdim9th** [comment]
Here it is!! Has everything 1.7 had and then some! Respect! Recognize!

**Comment by BigHonkinDaddy** [comment]
Well done on this release!!  I've been using the new release now for a while and like its fixes and improvements...great job!
